website/tw/06-libc.md

```diff
@@ -83,18 +83,18 @@ void *memset(void *buf, char c, size_t n) {
 
 > [!TIP]
 >
-> `*p++ = c;` 這行程式碼同時進行了「指標解參考（dereferencing）」與「指標遞增」的操作。為了更清楚說明，它等價於：
+> `*p++ = c;` 這行程式碼同時進行了「指標解參考（dereferencing）」與「指標位移」。為了更清楚說明，它等價於：
 >
 > ```c
-> *p = c;    // 解參考指標，將 c 賦值給目前指向的記憶體位置
-> p = p + 1; // 指標前進到下一個位置
+> *p = c;    // Dereference the pointer
+> p = p + 1; // Advance the pointer after the assignment
```

Comment:
question: Is English easier to read? You may rewrite them whenever appropriate 👍

Reply:
Oops I missed your comment. Consistency is a good point. Let's keep using English then 👍